crates/nu-cli/src/completion/matchers/case_insensitive.rs

```diff
@@ -0,0 +1,12 @@
+use crate::completion::matchers;
+use unicase::UniCase;
+pub struct Matcher;
+
+impl matchers::Matcher for Matcher {
+    fn matches(&self, partial: &str, from: &str) -> bool {
+        let from_u_substring: UniCase<&str> =
+            UniCase::new(from[0..std::cmp::min(from.len(), partial.len())].into());
+        let partial_u: UniCase<&str> = UniCase::new(partial.into());
+        return from_u_substring == partial_u;
```

Comment:
CI is failing because the `return` here is unnecessary

```suggestion
        from_u_substring == partial_u
```

(note, you can catch these errors locally by installing `clippy`, which you should then be able to run with `cargo clippy`)